pkg/agent/openflow/feature_templates.go

```diff
@@ -0,0 +1,361 @@
+//         \               |                /                |                 \               /
+//          \              |               /                 |                  \             /
+//           \             |              /                  |                   \           /
+//      +------------------------------------+        +--------------+       +--------------------+
+//      |               stage                |        |     stage    |       |        stage       |
+//      +------------------------------------+        +--------------+       +--------------------+
+//              \                                          /    \                        /
+//               \                                        /      \                      /
+//                \                                      /        \                    /
+//              +------------------------------------------+    +--------------------------+
+//              |                 feature                  |    |          feature         |
+//              +------------------------------------------+    +--------------------------+
+//                                  |                    \                         /
+//                                  |                     \                       /
+//                                  |                      \                     /
+//                     +------------------------+        +-------------------------+
+//                     |       OVS pipeline     |        |       OVS pipeline      |
+//                     +------------------------+        +-------------------------+
+
+// Table in FlexiblePipeline is the basic unit to build OVS pipelines. It is the enhanced version of ofTable. A table
+// can be used by one or more active features. If a Table is used by any active features, then the Table will be used
+// to build OVS pipelines and its member struct ofTable will be initialized and realized to OVS. Otherwise, it will be
+// excluded from building OVS pipelines. Note that, the order of all stages in OVS pipelines is fixed, but the order of
+// Tables within a stage is determined by the Tables' priority. Table with a higher priority will be assigned with a
+// smaller tableID for its corresponding ofTable in the realized OVS pipelines. Within a stage, a packet should enter
+// the ofTable with the highest priority first.
+type Table struct {
+	name     string
+	priority uint8
+	stage    binding.StageID
+	ofTable  binding.Table
+}
+
+// How to add a new table?
+// - Select a target OVS pipeline, feature and stage.
+// - Define a struct Table with table name, stage and priority. Arrange the Table to the desired position with appropriate
+//   priority value. All existing Tables are located at file pkg/agent/openflow/pipeline.go. For example, there is a
+//   stage which has Table Alpha (priority 200) and Table Beta (priority 100) declared by another feature.
```

Comment:
better to say `Table named 'Alpha'` and `Table named 'Beta'`, and wrap below `Alpha` and `Beta` `Gamma` with single quotes.